README.md

```diff
@@ -124,6 +124,8 @@ options:
                          only generate code for tables matching all of these regular expressions
   --exclude-tables EXCLUDE-TABLES
                          don't generate code for tables matching any of these regular expressions
+  --rename-types RENAME-TYPE
```

Comment:
Technically, we're not renaming anything. We're choosing a different name to begin with.

```suggestion
  --type-names NAMES
```